fleetspeak/src/client/socketservice/socketservice.go

```diff
@@ -93,6 +93,10 @@ func (s *Service) Start(sc service.Context) error {
 
 func (s *Service) Stop() error {
 	close(s.stop)
+	err := s.l.Close()
```

Comment:
It feels a bit untidy to close the listener both here and when channelManagerLoop finishes, but it is probably harmless to double-close. (I think we can probably just remove the other call.)

Reply:
Done.